packages/ts-config/main.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "main-tsconfig",
+  "compilerOptions": {
+      "target": "ESNext",
+      "moduleResolution": "node",
+      "module": "esnext",
+      "outDir": "dist",
+      "incremental": true,
+      "composite": true,
+      "declaration": true,
+      "resolveJsonModule": true,
+      "removeComments": true,
+      "emitDecoratorMetadata": true,
+      "experimentalDecorators": true,
+      "esModuleInterop": true,
+      "allowSyntheticDefaultImports": true,
+      "skipLibCheck": true,
```

nit: you set `skipLibCheck: true`. This can usually be set to `false` to check type declarations in the libraries you use. If you encounter problems removing this parameter, leave it set to true.

```suggestion
      "skipLibCheck": false,
```

---

I believe it's unnecessery ❤️